components/Modals/SignUp.tsx

```diff
@@ -99,16 +99,24 @@ export default function LoginModal({ onClose, state }) {
 
         <HorizontalScrollView>
           {[1, 2, 3, 4, 5].map((i) => (
-            <Avatar
-              key={i}
-              avatar={`/assets/avatars/${i}.png`}
-              size={80}
-              onClick={() => setAvatar(i)}
```

Comment:
The `onClick` function needs to go on the button. With a screen reader, the user can select the button, but pressing it doesn't do anything.